tools/coverage/presubmit_test.go

```diff
@@ -33,70 +33,110 @@ const (
 	testPresubmitBuild = 787
 )
 
-func repoDataForTest() *githubPr.GithubPr {
-	ctx := context.Background()
-	log.Printf("creating fake repo data \n")
+func TestRunPresubmit(t *testing.T) {
+	tests := []struct {
+		name     string
+		ghClient *githubClient.GithubClient
+		want     bool
+		wantErr  bool
+	}{
+		{
+			"RunPresubmitWithFakeGithubClient",
+			githubFakes.FakeGithubClient(),
+			false,
+			false,
+		},
+		{
+			"RunPresubmitWithNoGithubClient",
+			githubFakes.FakeGithubClient(),
```

Comment:
Still have it here

Reply:
i.e. I think it should just be an empty string? No github mock?

Reply:
Nice catch. It should be nil